src/services/trip.service.js

```diff
@@ -83,6 +83,21 @@ class TripService {
   static createMultiCityTrip(newTrip) {
     return Trip.bulkCreate(newTrip);
   }
+
+  /**
+ *
+ *
+ * @static
+ * @param {item} trip user column to be updated
+ * @param {value} tripInfo to be updated
+ * @returns {updated} @memberof UserService
```

Comment:
Please change thos to member of Trip service

Reply:
Ok didi it